workflows/sars-cov-2-variant-calling/sars-cov-2-pe-illumina-artic-variant-calling/pe-artic-variation.ga

```diff
@@ -864,14 +864,10 @@
         },
         "18": {
             "annotation": "",
-            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/ivar_removereads/ivar_removereads/1.3.1+galaxy0",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/ivar_removereads/ivar_removereads/1.3.1+galaxy2",
             "errors": null,
             "id": 18,
             "input_connections": {
-                "amplicon_info": {
-                    "id": 3,
-                    "output_name": "output"
-                },
```

Comment:
Hmm, this changes WF behavior. `ivar_removereads` at the suggested wrapper version *can* compute amplicon info on the fly if the structure of its "input_bed" input allows. However, it also still accepts an amplicon_info dataset conditionall, and for backwards-compatibility that's the desired way to go (compare also the Changelog for https://github.com/galaxyproject/iwc/pull/82).

So somehow potentially disrupting changes like this one (where an input_connection gets dropped) should be flagged by the bot. Not exactly sure how at the moment.

Reply:
It should be possible to flag the changes, I'll have a look at it.

I hope this can be considered as a nice future enhancement and not something that needs to be implemented straightaway though.

Reply:
> I hope this can be considered as a nice future enhancement and not something that needs to be implemented straightaway though.

Absolutely, the suggestions are mostly really useful already!